### ISSUE TITLE
Add latest commit date to the list of projects

### ISSUE THREAD

**Issue by kbsali** [netlify/staticgen#210]
While not always relevant because the latest commit date might not reflect the activity of a project,
it still gives some insight on whether the project is still alive or abandoned.

**Comment by bdougie** [comment]
This is a great idea and would happily accept a PR on this.

**Comment by akanshgulati** [comment]
Shall I work on this @bdougie ?

**Comment by erquhart** [comment]
@akanshgulati if you're up for it, please do!

**Comment by akanshgulati** [comment]
I think we should use some sort of algorithm to check the activity of the project. For ex. a last commit in Readme.md is less relevant than a last PR merge for major release in project.

**Comment by erquhart** [comment]
Yeah, days since last release might be more meaningful, agreed. Not necessarily major, though.
